packages/maven/io.sentry/sentry-android-replay/7.12.0.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "io.sentry:sentry-android-replay",
+    "canonical": "maven:io.sentry:sentry-android-replay",
+    "version": "7.12.0",
+    "package_url": "https://search.maven.org/artifact/io.sentry/sentry-android-replay",
+    "repo_url": "https://github.com/getsentry/sentry-java/tree/main/sentry-android-replay",
+    "main_docs_url": "https://docs.sentry.io/platforms/android/session-replay/",
```

Comment:
Won't 404 after we merge: https://github.com/getsentry/sentry-docs/pull/10711/files